HanksMineralEmporium/Service/AuthenticationService/ AuthenticationService.cs

```diff
@@ -0,0 +1,103 @@
+using System;
+using HanksMineralEmporium.Core.UserManagement;
+using HanksMineralEmporium.Core.UserManagement.Exception;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.DependencyInjection;
+using System.Threading.Tasks;
+using System.Runtime.CompilerServices;
+
+
+namespace HanksMineralEmporium.Service.AuthenticationService
+{
+    
+    public class  AuthenticationService : IAuthenticationService
```

Comment:
The indentation in the class is super messy and makes it difficult to read. I would also add a doc comment to this class to explain that it is an implementation for IAuthenticationService.